test/integration/cases/chart-needs.sh

```diff
@@ -1,15 +1,17 @@
 if [[ helm_major_version -eq 3 ]]; then
+  chart_need_case_input_dir="${cases_dir}/chart-needs/input"
+  chart_need_case_output_dir="${cases_dir}/chart-needs/output"
```

Comment:
I renamed the `issue.455` directory to `chart-needs` so that the test data can be found in the directory named same as the test script's basename, so that every test script has its own testdata placed within the directory whose name is the same as the test script itself.
Probably it's more consistent and easy to understand this way?

Reply:
make sense